tools/pipelines/templates/include-telemetry-setup.yml

```diff
@@ -23,10 +23,17 @@ parameters:
 - name: officeFeedUrl
   type: string
 
+# If the template should try to checkout the repo. If the job that includes this template already has steps to check out
+# the repo for other purposes, this should be set to false.
+- name: isCheckoutNeeded
+  type: boolean
+  default: false
```

Comment:
Even though all but one of the places that use this template are passing `true`, I think `false` is a safer default.